AZ3166/src/cores/arduino/system/SystemFunc.cpp

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. 
+ 
+#include "mbed.h"
+#include "mico.h"
```

Comment:
Both mbed.h and mico.h are needed here ?

Reply:
good catch